cpanfile

```diff
@@ -19,6 +19,7 @@ requires 'File::Spec';
 requires 'File::Temp';
 requires 'File::Which';
 requires 'IO::Handle';
+requires 'IO::Scalar';
```

Comment:
I suspect we still the analogous change adding this to `openQA.spec`

Reply:
Correction: os-autoinst.spec

Reply:
I added `IO::Scalar` in `docker/travis_test/Dockerfile` in https://github.com/os-autoinst/openQA/pull/2003 which was already merged. Shell I add it also into `openQA.spec` (requires new pull request in openQA git).

Reply:
INSTALL.asciidoc refers to spec file in https://build.opensuse.org/package/show/devel:openQA/os-autoinst, but I don't know where is git repository for it. Do I need to add it somewhere else?

Reply:
we have that spec only in obs

Reply:
It has been added there by me some time ago
https://build.opensuse.org/package/view_file/devel:openQA/os-autoinst/os-autoinst.spec?expand=1
https://build.opensuse.org/request/show/684257